src/VisualStudio/Core/Def/Implementation/PullMemberUp/MainDialog/PullMemberUpDialog.xaml

```diff
@@ -0,0 +1,189 @@
+             Width="500"
+             Title="{Binding ElementName=dialog, Path=PullMembersUpTitle}">
+    <Window.Resources>
+        <Thickness x:Key="okAndCancelButton">5, 3, 8, 4</Thickness>
+        <Thickness x:Key="selectDependantsButton">2, 4, 4, 2</Thickness>
+        <sys:Double x:Key="okAndCancelButtonWidth">50</sys:Double>
+        <sys:Double x:Key="okAndCancelButtonHeight">20</sys:Double>
+        <Style x:Key="LibraryTreeViewItemStyle"
+               TargetType="{x:Type TreeViewItem}">
+            <Setter Property="IsSelected"
+                        Value="{Binding IsSelected, Mode=TwoWay}" />
+            <Setter Property="FontWeight"
+                        Value="Normal" />
+            <Style.Triggers>
+                <Trigger Property="IsSelected" Value="True">
+                    <Setter Property="IsExpanded"
+                            Value="{Binding IsExpanded, Mode=TwoWay}" />
+                    <Setter Property="FontWeight" Value="Bold" />
+                </Trigger>
+            </Style.Triggers>
+        </Style>
+
+    </Window.Resources>
+    <Grid Margin="10,8,10,8">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="35*"/>
+            <RowDefinition Height="5*"/>
+            <RowDefinition Height="30*"/>
+            <RowDefinition Height="10*"/>
+        </Grid.RowDefinitions>
+
+        <StackPanel Grid.Row="0" Height="Auto" MinWidth="370" Margin="0,0,0,1">
+            <Label
+                x:Uid="Description"
+                x:Name="Description"
+                Content="  Select destination and members to pull up"/>
+            <Separator/>
+
+            <Label x:Name="TargetSelectionLabel"
+                   Content="Select destination type"/>
+
+            <TreeView x:Uid="PullTargetTreeView"
+                      x:Name="TargetMembersContainer"
+                      MinHeight="110"
+                      MinWidth="370"
+                      ScrollViewer.VerticalScrollBarVisibility="Visible"
+                      ItemsSource="{Binding TargetMembersContainer}"
+                      ItemContainerStyle="{StaticResource LibraryTreeViewItemStyle}"
+                      SelectedItemChanged="TargetMembersContainer_SelectedItemChanged">
+                <TreeView.ItemTemplate>
+                    <HierarchicalDataTemplate ItemsSource="{Binding Neighbours}"
+                                              DataType="{x:Type self:MemberSymbolViewModelGraphNode}">
+                        <StackPanel Orientation="Horizontal">
+                            <Image Source="{Binding MemberSymbolViewModel.Glyph}"/>
+                            <TextBlock Text="{Binding MemberSymbolViewModel.MemberName}"/>
+                        </StackPanel>
+                    </HierarchicalDataTemplate>
+                </TreeView.ItemTemplate>
+            </TreeView>
+        </StackPanel>
+
+        <Label Grid.Row ="1" x:Uid="MemberSelectionLabel" Content="Select members:"/>
```

Comment:
Use a `GroupBox` here (it helps conceptually/visually as well as for accessibility / reading the header when you tab into the content, the way ExtractInterface does).